netlify.toml

```diff
@@ -0,0 +1,4 @@
+[build]
```

Comment:
Having this file during the initial Netlify deployment caused the deployment to fail with the error: `11:22:40 AM: failed during stage 'building site': Deploy directory 'app/app/public' does not exist`. When I removed it, using the same directives defined in the UI, the deployments started to succeed.

Reply:
I think Netlify must have changed the syntax for the `publish` directive so that you don't have to repeat the `base` directory name. I ran into the same problem and was able to get it to work by removing the `app/` prefix from `publish` here: https://github.com/jeancochrane/national-parks-demo/commit/713ed1598fa668044e1e1c9e027d37d75215844e